docs/references/expo/offline-support.mdx

```diff
@@ -74,6 +74,8 @@ To enable offline support in your Expo app, follow these steps:
 
 ## How to handle network errors
 
+Until now, in Expo we used to mute these errors and developers could not handle them. This new experimental property prevents the muting and throws these network errors, allowing you to catch and handle them in your custom flows.
```

Comment:
This isn't necessary! We can remove this

Reply:
🤔  I added this because @royanger said we should add this context, if I understood correctly. ref: https://github.com/clerk/clerk-docs/pull/1806#discussion_r1887584952

Reply:
I see! I think he meant adding the context about "Clerk custom flows should throw errors if there is no internet connection". Let me rewrite it and get back to you

Reply:
take a look here - [aa9af7a](https://github.com/clerk/clerk-docs/pull/1806/commits/aa9af7a0f8570316bd70e6baee482d4d998e9252)
I think is the context Roy wanted to see added! and it makes sense, I like it :)

Reply:
Thanks! Looks really great!